services/GovDeliveryService.ts

```diff
@@ -133,25 +143,60 @@ export default class GovDeliveryService implements MonitoredService {
     });
   }
 
+  private static listApis(user: GovDeliveryUser): string {
+    const apis = user.apiList;
+    return apis.reduce((apiList, api, idx) => {
+      const properName = APIS_TO_PROPER_NAMES[api];
+      if (idx === 0) {
+        return properName;
+      } else if (idx === apis.length - 1 && apis.length === 2) {
+        return `${apiList} and ${properName}`;
+      } else if (idx === apis.length - 1 && apis.length > 2) {
+        return `${apiList}, and ${properName}`;
+      }
+      return `${apiList}, ${properName}`;
+    }, '');
+  }
+
+  // GovDelivery is considered healthy if <insert criteria>
```

Comment:
Not really related to these changes, but should we consider removing this comment?

Reply:
Yeah, we're probably safe removing this